udata/core/organization/views.py

```diff
@@ -76,8 +76,8 @@ def get_context(self):
         if self.organization.deleted and not can_view.can():
             abort(410)
 
-        datasets = Dataset.objects(organization=self.organization).visible()
-        reuses = Reuse.objects(organization=self.organization).visible()
+        datasets = Dataset.objects(organization=self.organization).order_by('-temporal_coverage.end', '-metrics.reuses', '-metrics.followers').visible()
```

Comment:
I think this goes over the 80 characters line limit (same below).

Reply:
Yes you are right. Can I chain calls on multiple lines? If so 👍

Reply:
With a `\` before the break line, yes.

Reply:
https://atom.io/packages/flake8

Reply:
I believe this also require settings the proper index on dataset model